setup.py

```diff
@@ -60,6 +62,19 @@ def run(self):
         }),
     ]
 
+cmdclass = {'build_ext': BuildClibBeforeExt}
+if sys.version_info > (3,) and platform.python_implementation() == "CPython":
+    try:
+        import wheel.bdist_wheel
+    except ImportError:
+        pass
+    else:
+        class BDistWheel(wheel.bdist_wheel.bdist_wheel):
+            def finalize_options(self):
+                self.py_limited_api = "cp3{}".format(sys.version_info[1])
```

Comment:
Does this work in 3.10 as well? Not sure where things are at wrt `310` v `3_10`?

Reply:
not sure either, I don't know if it has been decided yet.  `wheel` and `pip` currently disagree on what it should be

Reply:
but it doesn't really matter until python3.10 is your minimum supported version